frontend/apps/reader/modules/readerfont.lua

```diff
@@ -287,10 +283,11 @@ function ReaderFont:onSetFontSize(new_size)
 end
 
 function ReaderFont:onSetLineSpace(space)
-    self.line_space_percent = math.min(200, math.max(50, space))
-    self.ui.document:setInterlineSpacePercent(self.line_space_percent)
+    space = math.max(50, math.min(space, 200))
+    self.configurable.line_spacing = space
```

Comment:
Just curious: is this re-update of the self.configurable property enough for the updated value (which in practice is the same as the current one already stored by ReaderConfig) to be reflected in the bottom menu ?
I mean: if your math.max/min were ensuring _other_ limits (ie. 110-120), would they reflect in the bottom widget (progress bar updated, later launch of NumberPicker from <kbd>...</kbd> showing that value)?

Reply:
Yes, the clamped value is applied, saved and the bottom menu progress bar and spinwidget show it.

Reply:
Directly when the bottom menu is still opened ? Or do you need to close & reopen it?

Reply:
On "Apply" the spinwidget is not updated (shows still unclamped value), the notification shows the clamped value applied, the bottom menu at this moment is hidden, then we close the spinwidget and the reappearing bottom menu shows actual (clamped) value.

Reply:
What when you don't use the spinwidget, but just tap on the left or right most progress bar squares?

Reply:
Thanks. So, all that sounds rather good to me :)
Dunno if it will be as clean and simple from some of the other bottom settngs (ie; view_mode page/scroll).

Reply:
That's why "step by step".

Reply:
Yes :) But good to try a second step with a more complex case, to be sure this dance will look good and end well :)